frontend/src/store/NotificationState.ts

```diff
@@ -3,27 +3,78 @@
     It is structured this way so it doesn't stack multiple messages with the same severity (by design).
     It is intended to be used in conjunction with the NotificationStack component.
 */
-import type { Severity } from "@/enum/SeverityEnum";
+
 import { ref } from "vue"
+import { isDesktop, screenSize } from "@/store/screenSizeState";
+import type { Severity } from "@/enum/SeverityEnum";
 
 const defaultNotification = {
     success: "",
     warn: "",
     error: "",
 };
 
+export const newMsgList = ref();
```

Comment:
and we can't use one reactive variable to support multiple notification message, in the case we have both successful and warning message, the auto adjustment on the notification message will fail
<img width="501" alt="Screen Shot 2023-11-14 at 11 04 54 AM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/f4094812-f9b3-46e3-8102-2e92a89271c3">

Reply:
Hello! I'll fix it. Thanks!

Reply:
we can remove these three new variables